packages/material-ui/src/DialogContentText/DialogContentText.d.ts

```diff
@@ -3,7 +3,7 @@ import { StandardProps } from '..';
 import { TypographyProps } from '../Typography';
 
 export interface DialogContentTextProps
-extends StandardProps<TypographyProps, DialogContentTextClassKey> {}
```

Comment:
This change is not related to this PR. It was changed automatically by Prettier.